requests/models.py

```diff
@@ -420,8 +420,12 @@ def prepare_body(self, data, files, json=None):
         length = None
 
         if not data and json is not None:
+            # urllib3 requires a bytes-like body. Python 2's json.dumps
+            # provides this natively, but Python 3 gives a Unicode string.
```

Comment:
Should be avoiding mentioning urllib3 in this part of the codebase. I'll correct tomorrow, if I remember.